.github/pull_request_template.md

```diff
@@ -12,4 +12,12 @@ Resolves #
 
 ### Tests
 
-<!-- If applicable. Have you tested this pull request? If so, how? -->
+- [ ] Chrome
```

Comment:
Might want to expand this to say:
```suggestion
- [ ] Chrome/Chromium-based browser
```

Reply:
Should Edge also be an option since we also publish to their store, or do you think Chromium based browser covers that?

Reply:
> Should Edge also be an option since we also publish to their store, or do you think Chromium based browser covers that?

Stick with Chromium.